Documentation/contributing/development/contributing_guide.rst

```diff
@@ -77,6 +77,13 @@ requirements have been met:
        Make sure to include a blank line in between commit title and commit
        description.
 
+   .. note:
+
+       To avoid having to type common parts of the commit message every time,
+       you can create a commit template in git. See ``commit.template`` in
+       `Customizing Git - Git Configuration
+       <https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration>`.
```

Comment:
:exploding_head: 

What's the use case that made you add this? I usually only have `Signed-off-by: ...` as a recurring line.

Reply:
I also only have `Signed-off-by: ...` as a recurring line, but I got bored of typing it and was too lazy to add an editor macro to type it for me.

Reply:
Then I'm not sure this is needed. You can use `git commit -s`.

Reply:
Ah-ha, thank you! I didn't know about `git commit -s`, and also couldn't find it in the existing documentation. I've updated this PR to add a note about `git commit -s` instead of `commit.template`.

Reply:
Nice! :ship: